hack/kube-api-linter/exceptions.yaml

```diff
@@ -19,3 +19,11 @@
   path: "staging/src/k8s.io/api/resource/"
 - text: "Conditions field in AllocatedDeviceStatus has incorrect tags, should be: `json:\"conditions,omitempty\" patchStrategy:\"merge\" patchMergeKey:\"type\" protobuf:\"bytes,5,rep,name=conditions\"`"
   path: "staging/src/k8s.io/api/resource/"
+
+# OptionalOrRequired is being enabled over time. For now, each API group should be added to this list until we comb through each group and fix the missing tags.
+- text: "must be marked as optional or required"
+  path: "staging/src/k8s.io/api/(admission|apidiscovery|apps|authentication|authorization|autoscaling|batch|certificates|core|discovery|events|extensions|flowcontrol|imagepolicy|networking|node|policy|rbac|resource|scheduling|storage)"
```

Comment:
Will remove these 1x1 when we have the time to tackle the issues

Enumerated the count per group over https://github.com/kubernetes/kubernetes/issues/134671#issuecomment-3415486017